main.tf

```diff
@@ -84,9 +84,11 @@ resource "aws_rds_cluster" "default" {
   vpc_security_group_ids              = var.security_groups
   snapshot_identifier                 = var.snapshot_identifier
   storage_encrypted                   = var.storage_encrypted
+  kms_key_id                          = var.kms_key_id
```

Comment:
Does this need to be a ternary to handle storage_encrypted == true but kms_key_id == "" or should the var default to aws/rds